src/assets/css/_sass/components/_home-made-local.scss

```diff
@@ -4,7 +4,7 @@
   color: $color-text--dark;
   padding-top: 120px;
   height: 1200px;
-  background-image: url('/assets/images/_MG_0777.jpg');
+  background-image: linear-gradient(rgba(255,255,255,0.7), transparent, transparent), url("/assets/img/wood-3241848_1920.jpg");
```

Comment:
Sucks that the image is not predominantly white like the mocks. It might be nice to change the text to white here, and the gradient to a dark color.

Reply:
This is a placeholder image, so we're fine for now. I'll figure out a better image